rules-topics/11_ritual-magic.md

```diff
@@ -23,52 +23,55 @@ Following are the Standard Library, and other Rituals for which the Scroll must
 ### Rank I
 
 #### Elemental Shift
->This ritual removes “Magic Missile” as an option for an arcane magic user at every level it is offered and replaces them with one of either:
-> * Flame Bolt - “I conjure a flame bolt &lt;number>.”
-> * Frost Bolt - “I conjure a frost bolt &lt;number>.”
-> * Shock Bolt - “I conjure a shock bolt &lt;number>.”
-> * Acid Bolt  -  “I conjure a stone bolt &lt;number>.”
+This ritual removes “Magic Missile” as an option for an arcane magic user at every level it is offered and replaces them with one of either:
```

Comment:
I missed updating this from Magic Missile to Mana Bolt when we made that change.